docs/content/administration/command-line.en-us.md

```diff
@@ -95,7 +94,7 @@ Admin operations:
       - Options:
         - `--username value`, `-u value`: Username. Required.
         - `--password value`, `-p value`: New password. Required.
-        - `--must-change-password`: If provided, the user is required to choose a new password after the login. Optional.
+        - `--must-change-password`: The user is required to set a new password after the login, default: true. It could be disabled by `--must-change-password=false`.
```

Comment:
```suggestion
        - `--must-change-password`: The user must set a new password after login, default: true. Can be disabled using `--must-change-password=false`.
```

Reply:
`Can be disabled using ...` doesn't seem valid English 🤣 

I think it should be "be disabled by something"

Reply:
GPT also agrees with me:

![image](https://github.com/go-gitea/gitea/assets/2114189/371e8759-0f19-4a0d-b6b1-a7f7e306de49)